app/controllers/campaigns_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CampaignsController < APIController
+  def index
+    authorize Campaign
+    render json: Campaign.all
+  end
```

Comment:
I don't believe we need this index action. It's possible we'll do filtering on it, but I'm nervous about forgetting this endpoint exists.